templates/feature/project.config.js

```diff
@@ -93,4 +93,22 @@ module.exports = {
    *
    */
   customFilters: {},
+  /**
+   * Where custom settings for parsing graphics can be added.
+   *
+   * appleNewsIgnore
+   * Some graphics are too dynamic to be accurately captured in a screenshot.
+   * Those graphics shouldn't be considered for platforms like Apple News.
+   * Paths are relative to the build folders.
+   *
+   * Example:
+   * appleNewsIgnore: [
+   *  'complex-graphic-folder/index.html',
+   *  'some-other-folder',
+   * ],
+   *
+   */
+  parserOptions: {
```

Comment:
@mandicai, that's correct! Since `app/bananas.html` ultimately compiles to `/dist/bananas/index.html`, it'll skip that bananas folder per this [line](https://github.com/texastribune/data-visuals-create/blob/5261bbe2e18d509bb77f9564904571aadb57be46/templates/__common__/config/tasks/graphics-meta.js#L245). Alternatively, adding `bananas/index.html` to that project.config array would do the same.